README.md

```diff
@@ -71,7 +71,8 @@ matching route is found in the proxy table:
     -V, --version                    output the version number
     --ip <ip-address>                Public-facing IP of the proxy
     --port <n> (defaults to 8000)    Public-facing port of the proxy
-    
+    --storage-command <script>       The optional external executable to use for persistence
```

Comment:
I'm wondering if we can come up with a more specific name here than ``--storage-command``.

One suggestion:
``--persistent-storage <path>    Path to executable for persistent storage of route table (optional)``